ts/src/idl.ts

```diff
@@ -12,6 +12,11 @@ export type Idl = {
   events?: IdlEvent[];
   errors?: IdlErrorCode[];
   constants?: IdlConstant[];
+  metadata?: IdlMetadata;
+};
+
+export type IdlMetadata = {
+  address: string;
```

Comment:
This field is purposefully not defined and so we should probably make this `IdlMetadata = any` for now.